frontend/packages/dev-console/src/components/import/image-search/ImageStreamDropdown.tsx

```diff
@@ -13,6 +13,7 @@ const ImageStreamDropdown: React.FC<{ disabled?: boolean; formContextField?: str
   formContextField,
 }) => {
   const { t } = useTranslation();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why is this needed here?

Reply:
I will remove this and see what happens.